blueprint/reviews/review-likes.mson

```diff
@@ -0,0 +1,15 @@
+
+## reviewLikes (object)
++ id: 1 (required)
++ type: reviewLikes (required)
++ links (object)
```

Comment:
do you mean required per Kitsu.io or by JSON API standards? Because the later states it is optional.
<img width="729" alt="screen shot 2017-06-12 at 10 24 57" src="https://user-images.githubusercontent.com/6705325/27025123-9e64b5e2-4f59-11e7-8cfb-962f306866c8.png">

just making sure.